.env.example

```
@@ -11,5 +11,9 @@ NEXT_PUBLIC_APP_URL=""
 BETTER_AUTH_SECRET=""
 
 # Logo.dev Keys
-NEXT_PUBLIC_LOGO_DEV_TOKEN=
-LOGO_DEV_TOKEN=
\ No newline at end of file
+NEXT_PUBLIC_LOGO_DEV_TOKEN=""
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)